### DISCUSSION TITLE
optimal Nvidia gpu and Ip camera

### DISCUSSION THREAD

**Discussion by Touville01** [roboflow/supervision#1279] [Q&A] [answered]
Hi everyone, I'm working on this project in order to do some  real time demo, and I need some advices on hardware.
Currently I'm using a windows thinkbook and I'm planning to do real time demo with a remote camera.

I'm considering purchasing a Nvidia graphics card and an ip camera that supports differents communications protocols, especially rtsp, with customizable settings for various uses cases.

Has anyone here done similar real time demos ? Could you recommend the bests models for these cases (count people, time, tracking), in terms of quality price? Based on experiences and feedback, what would be the optimal setup for achieving ideal performance?

Thank you !

**Comment by LinasKo** [comment]
Hi @Touville01 👋 

Unfortunately, that's outside our area of focus. A good answer requires a deeper conversation, digging into requirements specific to your use case, and a survey of best practices used by our current customers. None of which we, open-source developers, have the time for.

Best of luck with your project - shame I couldn't help this time.

**Comment by rolson24** [comment, marked as answer]
Hi @Touville01!

I would recommend starting with deciding what object detection model you want to use, and then consider how much VRAM that model will require. After you have decided that, you can look at FP16 throughput performance for different GPU's (benchmarks all over the internet) and decide which gpu to get based on performance and price.  The main bottleneck is the object detection model which will be the only thing running on the gpu. As for the camera it is hard to make recommendations without knowing your specific usecase.

If you are using YOLOv8, I would recommend joining the Ultralytics discord, as there is an active community of people working on computer vision projects who might have some good recommendations.
